docs/concepts/registries.md

```diff
@@ -43,6 +43,15 @@ public void registerBlocks(RegistryEvent.Register<Block> event) {
 }
 ```
 
+### Registries that aren't Forge Registries
+
+Due to some peculiarities of vanilla code, not all registries are wrapped by Forge. These can be static registries, like `RecipeType`, which are safe to use. There are also dynamic registries, like `ConfiguredFeature` and some other worldgen registries, which are typically represented in JSON. These objects should only be registered this way if there is another registry object that requires it. 
+
+Without the benefit of registry objects, we have to be careful of when we do the registration and how.
```

Comment:
Try to avoid using first person pronouns in sentences. This is not heavily enforced, but I would eventually like to transition the documentation away from making it seem like a narrative banter.